src/client/src/Include_i.h

```diff
@@ -280,6 +280,9 @@ typedef struct __KinesisVideoClient {
     // ID for timer created to wake and check if streams have incoming data
     UINT32 timerId;
 
+    // Retry strategy for the client and all the streams under it
+    KVSRetryStrategy kVSRetryStrategy;

```

Comment:
Odd naming. recommend simply retryStrategy

Reply:
Intentionally added KVS as a prefix to avoid any potential conflicts. retryStrategy sounds very common.

Reply:
nit: KVS is an acronym so making the variable name `kVSRetryStrategy` looks weird, how about `kvsRetryStrategy` ?

Reply:
Sure. Changed `KVSRetryStrategy` type to `KvsRetryStrategy` and all variables to `kvsRetryStrategy`.
It was a sweeping change. You would see a larger diff now.